frontend/integration-tests/tests/kubevirt/vm.overview.scenario.ts

```diff
@@ -122,73 +115,72 @@ describe('VM Services', () => {
       await isLoaded();
     });
   });
-});
-
-describe('Test vm overview on project overview page', () => {
-  const exposeServices = new Set<any>();
-
-  beforeEach(async() => {
-    await browser.get(`${appHost}/overview/ns/${testName}`);
-    await isLoaded();
-  });
-
-  it('Check vm details on project overview', async() => {
-    expect(vmView.statusIcon(vmView.statusIcons.off).isPresent()).toBeTruthy();
-    await itemVirtualMachine.click();
-    await isLoaded();
-    await detailViewAction('Start', true);
-    await browser.wait(until.presenceOf(vmView.statusIcon(vmView.statusIcons.running)), VM_BOOTUP_TIMEOUT);
-
-    expect(vmView.vmDetailDesc(testName, vmName).getText()).toEqual(newVMDescription);
-    expect(vmView.vmDetailOS(testName, vmName).getText()).toEqual(basicVmConfig.operatingSystem);
-    expect(vmView.statusIcon(vmView.statusIcons.running).isPresent()).toBeTruthy();
-    expect(vmView.vmDetailTemplate(testName, vmName).getText()).toEqual('openshift/rhel7-generic-small');
-    expect(vmView.vmDetailNamespace(testName, vmName).$('a').getText()).toEqual(testName);
-    expect(vmView.vmDetailBootOrder(testName, vmName).getText()).toEqual(['rootdisk', 'nic0', 'cloudinitdisk']);
-    expect(vmView.vmDetailFlavorDesc(testName, vmName).getText()).toEqual('2 CPU, 4G Memory');
-    expect(vmView.vmDetailIP(testName, vmName).getText()).not.toEqual(emptyStr);
-    expect(vmView.vmDetailHostname(testName, vmName).getText()).toEqual(vmName);
-    expect(vmView.vmDetailPod(testName, vmName).$('a').getText()).toContain('virt-launcher');
-    expect(vmView.vmDetailNode(testName, vmName).$('a').getText()).not.toEqual(emptyStr);
 
-    await asyncForEach(exposeServices, async(srv) => {
-      expect(vmView.vmDetailService(testName, srv.exposeName).getText()).toEqual(srv.exposeName);
+  describe('Test vm overview on project overview page', () => {
+    beforeEach(async() => {
+      await browser.get(`${appHost}/overview/ns/${testName}`);
+      await isLoaded();
     });
-  });
 
-  it('Click vm link', async() => {
-    virtualMachineLink(0).click();
-    expect(browser.getCurrentUrl()).toEqual(`${appHost}/k8s/ns/${testName}/virtualmachines/${vmName}`);
-  });
+    xit('BZ(1714625) Check vm details on project overview', async() => {
```

Comment:
xit?

Reply:
Skipped due to BZ 1714625

Reply:
ok, can we have a TODO here at least please? with a reason of skipping this...

Reply:
It's actually fairly visible in the test run report, however, now I noticed that another spec depends on this one, which is not good and should be addressed... So I'll let it fail for now
![skipped_spec](https://user-images.githubusercontent.com/32546791/58535197-a5c69d00-81ed-11e9-8f2e-420fea5d8c6a.png)

Reply:
> It's actually fairly visible in the test run report,

ah, ok didn't know that

> however, now I noticed that another spec depends on this one, which is not good and should be addressed... So I'll let it fail for now

ok, I will let it up to you